src/ripple/beast/core/SystemStats.h

```diff
@@ -30,10 +30,6 @@
 namespace beast
 {
 
-//==============================================================================
-/** Returns the host-name of the computer. */
-std::string getComputerName();
-
 //==============================================================================
 /** Returns a backtrace of the current call-stack.
     The usefulness of the result will depend on the level of debug symbols
```

Comment:
slightly confusing, but `getStackBacktrace()` was already removed here:  https://github.com/ripple/rippled/pull/2376  ...which is soon to be merged. There's probably a case to be made for merging to two PRs, but what I can do is just rebase and remove at that point.

Reply:
There's definitely a case for merging the two Prs so we can actually delete SystemStats.h.  Otherwise a rebase has the potential for leaving the file hanging around and empty.